pandas/_typing.py

```diff
@@ -90,18 +90,29 @@
     from typing import SupportsIndex
 
     if sys.version_info >= (3, 10):
+        from typing import Concatenate  # pyright: ignore[reportUnusedImport]
+        from typing import ParamSpec
         from typing import TypeGuard  # pyright: ignore[reportUnusedImport]
```

Comment:
I'm surprised isort allows that, would have expected that you need:

```py
# or with only one ignore, not sure what pyright expects
from typing import (
    Concatenate,  # pyright: ignore[reportUnusedImport]
    ParamSpec,  # pyright: ignore[reportUnusedImport]
    TypeGuard,  # pyright: ignore[reportUnusedImport]
)
```

Reply:
Feel free to keep as-is as long as isort+ruff do not complain :)

Reply:
> I'm surprised isort allows that,

Yeah me too, your suggestion is what I originally had to be honest. isort is what formatted it to what we see now.